viadot/examples/sap_rfc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM viadot:dev
+FROM viadot:2.0a11
```

Comment:
This should be 2.0a12 in fact as a11 version doesn't have `SAPRFC` yet (once this is merged I'll publish this new release).

Reply:
```suggestion
FROM viadot:2.0a12
```